Makefile

```diff
@@ -82,19 +82,23 @@ ifneq ($(V),1)
   # Do not print "Entering directory ...".
   MAKEFLAGS += --no-print-directory
   # Redirect stdout/stderr for chatty tools
-  NOOUT = 1> /dev/null 2> /dev/null  
+  NOOUT = 1> /dev/null 2> /dev/null
 endif
 
-FREERTOS_D_FILES       = $(FREERTOS_C_FILES:.c=.d)
-FREERTOS_O_FILES       = $(FREERTOS_C_FILES:.c=.o)       $(FREERTOS_S_FILES:.s=.o)
-STM32F0_PERIPH_D_FILES = $(STM32F0_PERIPH_C_FILES:.c=.d)
-STM32F0_PERIPH_O_FILES = $(STM32F0_PERIPH_C_FILES:.c=.o) $(STM32F0_PERIPH_S_FILES:.s=.o)
-STM32F4_PERIPH_D_FILES = $(STM32F4_PERIPH_C_FILES:.c=.d)
-STM32F4_PERIPH_O_FILES = $(STM32F4_PERIPH_C_FILES:.c=.o) $(STM32F4_PERIPH_S_FILES:.s=.o)
-STM32_USB_D_FILES      = $(STM32_USB_C_FILES:.c=.d)
-STM32_USB_O_FILES      = $(STM32_USB_C_FILES:.c=.o)      $(STM32_USB_S_FILES:.s=.o)
-APP_D_FILES            = $(APP_C_FILES:.c=.d)
-APP_O_FILES            = $(APP_C_FILES:.c=.o)            $(APP_S_FILES:.s=.o)
+STM32F0_DRIVERS_D_FILES = $(STM32F0_DRIVERS_C_FILES:.c=.d)
+STM32F0_DRIVERS_O_FILES = $(STM32F0_DRIVERS_C_FILES:.c=.o)
```

Comment:
Maybe do the .s -> .o rule here as well? Even if you have no assembly now, it keeps people from being surprised later should they happen to add some. ("Why aren't these files getting built? What the heck?") Make sure to init STM32F0_DRIVERS_S_FILES to empty, though.